MOCC/src/configurations/Comm-Layer/main.py

```diff
@@ -0,0 +1,102 @@
+import sqlite3
+from flask import Flask, request, jsonify
+import datetime
+import os
+from flasgger import Swagger
+
+
+
+if 'tango.db' not in os.listdir('.'):
+    conn = sqlite3.connect('tango.db', check_same_thread=False)
+    lookup_table = conn.cursor()
+    lookup_table.execute('''CREATE TABLE lookup(timestamp datetime,tango_addr varchar,ip_addr varchar)''')
+    conn.commit()
+    conn.close()
+
+app = Flask(__name__)
+swagger = Swagger(app)
+
+@app.route('/save/<path:tango_addr>/', methods=['GET'])
+def save(tango_addr):
+    """Endpoint to map a Tango device address to the sender's IP address
+    ---
+    tags:
+      - save
+    parameters:
+      - name: tango_addr
+        in: path
+        type: string
+        required: true
+    responses:
+      200:
+        description: Device address successfully saved.
+        type: string
+      500:
+        description: Internal server error
+        type: string
+    """
+    conn = sqlite3.connect('tango.db', check_same_thread=False)
+    lookup_table = conn.cursor()
+    ip_addr = request.remote_addr
+    try:
+        values = (datetime.datetime.now(), tango_addr, ip_addr)
+        lookup_table.execute('INSERT INTO lookup VALUES (?,?,?)', values)
+        conn.commit()
+    except Exception as e:
+        print e
```

Comment:
Sorry, I forgot to tell you that we want to develop on Python 3.6

Reply:
Oh OK. Will change it and add the commit.